packages/FAQ/docs/how-does-it-work.md

```diff
@@ -0,0 +1,7 @@
+---
+title: How Does it Work?
```

Comment:
Is this WIP? Feels like the answer to this question needs to be more substantial than "go to the website"

Reply:
Yes, I may need your help here.